### PR TITLE
fix: layout issue when stacking

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/7655724502), [Test report](https://leather-wallet.github.io/playwright-reports/fix/stacking-layout)

Ref #4851

image

Fixes layout issue and makes some minor visual improvements.

### CODE REVIEW

File: src/app/components/caption-dot-separator.tsx [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-import { BoxProps, styled } from 'leather-styles/jsx';
+import { Circle, CircleProps } from 'leather-styles/jsx';
 
-function CaptionSeparatorDot(props: BoxProps) {
-  return (
-    <styled.span color="accent.text-subdued" fontSize="10px" {...props}>
-      •
-    </styled.span>
-  );
+export function CaptionSeparatorDot(props: CircleProps) {
```
Comment (fbwoolf) [inline src/app/components/caption-dot-separator.tsx]: Move to `ui` components?

**Review by fbwoolf — APPROVED** [review]

**Review by pete-watters — APPROVED** [review]

File: src/app/components/caption-dot-separator.tsx [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
+  return <Circle display="inline-flex" backgroundColor="currentColor" size="3px" {...props} />;
```
Comment (edgarkhanzadian) [inline src/app/components/caption-dot-separator.tsx]: We also have that symbol `•` in `src/app/components/crypto-assets/components/asset-caption.tsx`, wondering if using CaptionSeparatorDot there would keep these circle separators more consistent.

**Review by edgarkhanzadian — APPROVED** [review]

File: src/app/components/caption-dot-separator.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/app/components/caption-dot-separator.tsx]: Will migrate & write stories 📖

File: src/app/components/caption-dot-separator.tsx (hunk 2 above)
Comment (kyranjamie) [reply src/app/components/caption-dot-separator.tsx]: done

**Pull request merged**